data/mods/Aftershock/maps/mapgen/augmentation_clinic.json

```diff
@@ -0,0 +1,259 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "afs_augmentation_clinic_n1" ],
+    "weight": 200,
```

Comment:
Nitpick, but you don't need to add this field if there is only one variant of this particular om_terrain.